rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -1341,70 +1220,99 @@ public BooleanWrapper createNodeSource(String nodeSourceName, String infrastruct
         }
     }
 
-    protected BooleanWrapper createNodeSource(NodeSourceData nodeSourceData, boolean nodesRecoverable) {
-        String nodeSourceName = nodeSourceData.getName();
+    public void createNotActiveNodeSource(NodeSourceData persistedNodeSource) {
```

Comment:
I am not sure it should be public

Reply:
I can leave NodesRecoveryManager in the same package and leave package-protected ?